storybook/pages/SwapInputPanelPage.qml

```diff
@@ -0,0 +1,182 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+import shared.stores.send 1.0
+
+import AppLayouts.Wallet.stores 1.0
+import AppLayouts.Wallet.panels 1.0
+import AppLayouts.Wallet.controls 1.0
+
+import Models 1.0
+import Storybook 1.0
+
+import SortFilterProxyModel 0.2
+
+SplitView {
+    id: root
+
+    Logs { id: logs }
+
+    QtObject {
+        id: d
+
+        readonly property string networksChainsCurrentlySelected: {
+            // let supportedNwChains = []
+            // for (let i = 0; i< networksRepeater.count; i++) {
+            //     if (networksRepeater.itemAt(i).checked && networksRepeater.itemAt(i).visible)
+            //         supportedNwChains.push(networksRepeater.itemAt(i).chainID)
+            // }
+            // return supportedNwChains.join(":")
+            return "1:5:10:420:421613"
+        }
+
+        property WalletAssetsStore walletAssetStore: WalletAssetsStore {
+            assetsWithFilteredBalances: SubmodelProxyModel {
+                sourceModel: d.walletAssetStore.groupedAccountsAssetsModel
+                submodelRoleName: "balances"
+                delegateModel: SortFilterProxyModel {
+                    sourceModel: submodel
+                    filters: FastExpressionFilter {
+                        expression: {
+                            d.networksChainsCurrentlySelected
+                            return d.networksChainsCurrentlySelected.split(":").includes(model.chainId+"")
+                        }
+                        expectedRoles: ["chainId"]
+                    }
+                }
+            }
+        }
+
+        property TransactionStore store: TransactionStore {
+            walletAssetStore: d.walletAssetStore
+            //showCommunityAssetsInSend: false // FIXME in reality, this is in TokensStore
+        }
+    }
+
+    Rectangle {
+        SplitView.fillWidth: true
+        SplitView.fillHeight: true
+        color: Theme.palette.baseColor3
+
+        Item {
+            width: 492
+            height: payPanel.height + receivePanel.height + 4
+            anchors.centerIn: parent
+
+            SwapInputPanel {
+                id: payPanel
+                anchors {
+                    left: parent.left
+                    right: parent.right
+                    top: parent.top
+                }
+                store: d.store
+
+                inputTokenKey: ctrlFromTokensKey.text
+                inputTokenAmount: ctrlFromTokenAmount.text
+
+                swapSide: SwapInputPanel.SwapSide.Pay
+                swapExchangeButtonWidth: swapButton.width
+            }
+
+            SwapInputPanel {
+                id: receivePanel
+                anchors {
+                    left: parent.left
+                    right: parent.right
+                    bottom: parent.bottom
+                }
+                store: d.store
```

Comment:
I think we can start to use the SwapModalAdaptor and Store already as it is already merged.

Reply:
Right, I felt that was the ultimate idea but I thought I'd rather leave it up to you for later because not everything that I need for these subcomponents is already available from the new SwapModalAdaptor/Store

Reply:
I mean not against it, we can definitely improve that in the process of integrating this into the SwapModal